src/test/java/opgg/weba/JamPick/controller/RouletteRecControllerTest.java

```diff
@@ -48,6 +49,36 @@
     @Autowired
     RouletteRecRepository rouletteRecRepository;
 
+    @Autowired EntityManager em;
+
+    @BeforeEach
+    public void before() {
+
+        IndieApp indieApp1 = new IndieApp();
```

Comment:
`RoutletteRecRepositoryTest.java` file has same code. Make this code to function or class is better to me.

Reply:
It's okay that resolve this other task. Resolve this conversation and merge. Good job.